core/inspector.rs

```diff
@@ -365,6 +365,9 @@ impl JsRuntimeInspector {
             w.poll_state = PollState::Parked;
             w.parked_thread.replace(thread::current());
           }
+          PollState::Parked => {
+            w.parked_thread.replace(thread::current());
+          }
```

Comment:
This is still concerning to me and I would like to get to the bottom of this problem before landing the PR

Reply:
Addressed by https://github.com/denoland/deno/pull/19519/commits/96f886735cfa7da11fbaffa32f1655b8340ad454. There was a redundant `run_event_loop()` from `cli/build.rs` for the already evaluated `deno_node` entrypoint. I suppose that led to the inspector being initialised to a parked state. Maybe it would be fine either way.